projects/objects/kitchen/components/protos/Worktop.proto

```diff
@@ -4,56 +4,41 @@
 # A customizable (size, texture, etc.) worktop.
 
 PROTO Worktop [
-  field SFVec3f    translation   0 0.71 0
-  field SFRotation rotation      0 1 0 0
-  field SFString   name          "worktop"
-  field SFVec3f    size          0.44 0.06 0.7           # Defines the size of the worktop.
-  field MFString   textureUrl    "textures/worktop.jpg"  # Defines the texture used for the worktop.
-  field SFVec2f    tileSize      0.7 0.7                 # Defines the size of texture used for the worktop.
+  field SFVec3f    translation       0 0.71 0
+  field SFRotation rotation          0 1 0 0
+  field SFString   name              "worktop"
+  field SFVec3f    size              0.4 0.06 0.7  # Defines the size of the worktop.
+  field SFFloat    mass              0             # Defines the mass of the wooden box in kg. A value smaller or equal to 0 remove the physics of the wooden box.
+  field SFFloat    IBLStrength       1             # Defines the strength of ambient lighting from the Background node.
```

Comment:
Those two fields are useless, instead an `appearance` field should be opened.
And this should be mentioned in the changelog.